Xamarin.Forms.Platform.Android/Renderers/TableViewModelRenderer.cs

```diff
@@ -29,41 +39,35 @@ public override int Count
 		{
 			get
 			{
-				var count = 0;
-
-				//Get each adapter's count + 1 for the header
-				ITableModel model = Controller.Model;
-				int section = model.GetSectionCount();
-				for (var i = 0; i < section; i++)
-					count += model.GetRowCount(i) + 1;
-
-				return count;
+				if (cellCache == null)
+					FillCache();
```

Comment:
You're doing this `if` logic every time before you use `cellCache`. I wonder if it should just be a private property with the logic inside the getter.